table/queries/derived_satellite_scans.sql

```diff
@@ -73,6 +73,21 @@ CREATE TEMP FUNCTION AnswersSignature(answers ANY TYPE) AS (
   ), ",")
 );
 
+# resolver_*_rate fields measure various error rates for each resolver across the whole scan
+# They should be a value from 0-1 calculated as
+# number_of_unique_domains_with_error_on_resolver_per_scan
+#  /total_number_of_unique_domains_on_resolver_per_scan
+# resolvers which consistantly return too many errors are filtered out.
+CREATE TEMP FUNCTION BadResolver(
+  resolver_connect_error_rate FLOAT64,
+  resolver_invalid_cert_rate FLOAT64,
+  resolver_non_zero_rcode_rate FLOAT64
+) AS (
+  resolver_connect_error_rate > .75 OR
```

Comment:
Could you add a comment explaining that this is timeout?

Reply:
done